pkg/runner/local_docker.go

```diff
@@ -178,10 +177,14 @@ func (*LocalDockerRunner) Run(input *api.RunInput) (*api.RunOutput, error) {
 		containers = append(containers, res.ID)
 	}
 
+	if !cfg.KeepContainers {
+		defer deleteContainers(cli, log, containers)
```

Comment:
Go is unhappy here - https://github.com/ipfs/testground/pull/126#pullrequestreview-313209888

```suggestion
		defer func () {
  		err := deleteContainers(cli, log, containers)
  		if err != nil {
  		   // abort here?
  		}
		}
```
Or is there other best practice to silence this lint error?

Reply:
In this case, we probably want to log.

Reply:
(fixed)